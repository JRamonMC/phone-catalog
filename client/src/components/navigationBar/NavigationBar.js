import React from "react";

const NavigationBar = () => {
  return (
    <div>
      <nav className="navbar navbar-expand-lg navbar-dark bg-primary">
        <div className="container-fluid">
          <a className="navbar-brand" href="/">
            Phones Catalogue
          </a>
          <a className="navbar-brand" aria-current="page" href="/add">
            Add new phone
          </a>
        </div>
      </nav>
    </div>
  );
};

export default NavigationBar;
