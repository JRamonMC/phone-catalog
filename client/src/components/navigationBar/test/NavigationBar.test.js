import React from "react";
import NavigationBar from "./../NavigationBar";
import { shallow } from "enzyme";

describe("Navigation Bar test", () => {
  it("Navigation Bar render", () => {
    const wrapper = shallow(<NavigationBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
