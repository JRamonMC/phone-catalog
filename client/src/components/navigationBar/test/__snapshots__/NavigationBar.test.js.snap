// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Bar test Navigation Bar render 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-primary"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand"
        href="/"
      >
        Phones Catalogue
      </a>
      <a
        aria-current="page"
        className="navbar-brand"
        href="/add"
      >
        Add new phone
      </a>
    </div>
  </nav>
</div>
`;
